### DISCUSSION TITLE
unused constant FEED_PRECISION in DSCEngine.sol?

### DISCUSSION THREAD

**Discussion by lukekorkowski** [Cyfrin/foundry-full-course-cu#861] [Q&A]
Am I missing something, or is the constant FEED_PRECISION in DSCEngine.sol in lesson 12 never actually used? It's declared and defined on line 77 [here](https://github.com/Cyfrin/foundry-defi-stablecoin-f23/blob/main/src/DSCEngine.sol), but I don't see it being employed anywhere.

What am I missing?

**Comment by 0xusmanf** [comment]
Yes you are right it is not being used anywhere.

**Reply by lukekorkowski** [reply]
Thanks for confirming!

**Comment by theirrationalone** [comment]
Yeah! You should make a PR. ;`)
